book/src/developers/contributing/releases/deployment.md

```diff
@@ -53,6 +53,9 @@ This step directs Ansible to use the current master version of trin. Read [about
 - Make sure you've pulled the latest master branch of the deployment scripts, to include any recent changes: `git pull origin master`
 - Go into Portal section of Ansible: `cd portal-network/trin/ansible/`
 - Run the deployment: `ansible-playbook playbook.yml --tags trin`
+- Run Glados deployment
```

Comment:
Could you add a little more detail, what exactly will this restart? I'm guessing just the `trin` instance that glados uses, and not also the glados processes themselves? If I'm right, then something like ...
`Glados: To update the Trin instance that Glados uses:` would be great

Reply:
This is to run a glados deployment. Glados can run both Trin and Fluffy. So this command would update the glados crates and update whichever Portal client is configured